data/json/overmap_specials.json

```diff
@@ -534,6 +534,26 @@
         "flags" : [ "CLASSIC" ]
     },{
         "type" : "overmap_special",
+        "id" : "Office Tower 2.0",
```

Comment:
OK, I see that it worked OK in the original tower, but we'd like to keep spaces out of id strings where possible.